datasets/libsvm.py

```diff
@@ -25,6 +26,10 @@ def get_data(self):
 
         X, y = fetch_libsvm(self.dataset)
 
+        if self.dataset == "YearPredictionMSD":
+            scaler = StandardScaler()
+            X = scaler.fit_transform(X)
```

Comment:
before merging this will this break the figure on Lasso in the paper as we have a condition with Raw (unscaled data and no fit_intercept)

Reply:
In logreg yes but not in Lasso